src/constants/skills.js

```diff
@@ -536,7 +556,12 @@ const icons = {
   postman: postman,
   erlang: erlang,
   sqlite: sqlite,
-  middleman: middleman
+  middleman: middleman,
+  JAM_stack: jamStack,
+  MEAN_stack: meanStack,
+  MERN_stack: mernStack,
+  LAMP_stack: lampStack,
```

Comment:
remove the comma here, after lampStack, that is why the build i failing i think

Reply:
No it didn't affect but got the bug. Thank you @Aditya-ds-1806

Reply:
oh okay😅